deeppavlov/models/torch_bert/torch_bert_squad.py

```diff
@@ -150,24 +149,26 @@ def __call__(self, features: List[InputFeatures]) -> Tuple[List[int], List[int],
             predictions: start, end positions, start, end logits positions
 
         """
-        input_ids = [f.input_ids for f in features]
-        input_masks = [f.attention_mask for f in features]
-        input_type_ids = [f.token_type_ids for f in features]
+        _input = {}
+        for elem in ['input_ids', 'attention_mask', 'token_type_ids']:
+            _input[elem] = [getattr(f, elem) for f in features]
 
-        b_input_ids = torch.cat(input_ids, dim=0).to(self.device)
-        b_input_masks = torch.cat(input_masks, dim=0).to(self.device)
-        b_input_type_ids = torch.cat(input_type_ids, dim=0).to(self.device)
+        for elem in ['input_ids', 'attention_mask', 'token_type_ids']:
```

Comment:
Hard-coding this is not a good idea since some models (roberta, for example) don't use token_type_ids.